src/components/AvatarExamples.jsx

```diff
@@ -0,0 +1,46 @@
+import PropTypes from 'prop-types';
+
+const arrayOfAvatars = [
+  'https://robohash.org/udsbnudhfksuduhk.png?size=50x50&set=set1',
+  'https://robohash.org/autinvenrfssdfsu.png?size=50x50&set=set1',
+  'https://robohash.org/lsdkfjlskdjflask.png?size=50x50&set=set1',
+  'https://robohash.org/ksdjfosilsmncisk.png?size=50x50&set=set1',
```

Comment:
I'm thinking this array could likely be simpler in some way - it doesn't require much attention to read, but it's also a lot to look at.